target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4a-100m-intl-v2.dts

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an_xiaomi_mi-router-4.dtsi"
+
+/ {
+	compatible = "xiaomi,mi-router-4a-100m-intl-v2", "mediatek,mt7628an-soc";
+	model = "Xiaomi Mi Router 4A (100M International Edition V2)";
+
+	aliases {
+		led-boot = &led_power_yellow;
+		led-failsafe = &led_power_yellow;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_yellow;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_blue: power_blue {
+			label = "blue:power";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_yellow: power_yellow {
+			label = "yellow:power";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			label = "blue:wan";
```

Comment:
```suggestion
			color = <LED_COLOR_ID_BLUE>;
			function = LED_FUNCTION_WAN;
```

Reply:
Should I also change this symbols in 4a-100m, 4a-100m-intl and 4c dtses? Or it's ok if this device will use leds/common.h when other similar will not?

Reply:
If you like, please create another commit to convert them. The `label` property is deprecated. ref:
https://github.com/torvalds/linux/blob/610a9b8f49fbcf1100716370d3b5f6f884a2835a/Documentation/devicetree/bindings/leds/common.yaml#L54

Reply:
Done. Thank you!